services/docker-compose.devel.yml

```diff
@@ -63,6 +63,10 @@ services:
     volumes:
       - ./storage:/devel/services/storage
       - ../packages:/devel/packages
+    environment:
+      - SC_BOOT_MODE=debug-ptvsd-disable
+    ports:
+      - "3000:3000"
```

Comment:
Do we have any kind of criteria to assign port numbers?

Reply:
no, but I guess we can define some rules